server/chat-plugins/the-studio.ts

```diff
@@ -313,18 +316,16 @@ class RecommendationsInterface {
 	}
 
 	async render(rec: Recommendation, suggested = false) {
-		let videoInfo = null;
-		try {
-			videoInfo = await YouTube.getVideoData(rec.url);
-		} catch (e) {
-			throw new Chat.ErrorMessage(`Error while fetching recommendation URL: ${e.message}`);
-		}
 		let buf = ``;
 		buf += `<div style="color:#000;background:linear-gradient(rgba(210,210,210),rgba(225,225,225))">`;
 		buf += `<table style="margin:auto;background:rgba(255,255,255,0.25);padding:3px;"><tbody><tr>`;
-		if (videoInfo) {
-			buf += `<td style="text-align:center;"><img src="${videoInfo.thumbnail}" width="120" height="67" /><br />`;
-			buf += `<small><em>${!suggested ? `${Chat.count(rec.likes, "points")} | ` : ``}${videoInfo.views} views</em></small></td>`;
+		if (rec.videoInfo === undefined) {
+			rec.videoInfo = await YouTube.getVideoData(rec.videoInfo);
+			saveRecommendations();
+		}
```

Comment:
ESLint is flagging this. `rec.videoInfo` is guaranteed to be `undefined` here, but `await YouTube.getVideoData()` seems like it's guaranteed to fail?